### ISSUE TITLE
Alternative for Google Places Api

### ISSUE THREAD

**Issue by kpencak** [zJavaproject/DayTripper#4]

**Comment by kpencak** [comment]
![obraz](https://user-images.githubusercontent.com/33725834/64032797-7fe1c900-cb4b-11e9-9fea-74029790795d.png)

**Comment by kpencak** [comment]
I would go with Sygic places or here, both are good. Or we can choose both and mix it

**Comment by kopeadri** [comment]
I agree, Here has really good documentation and provides many detailed info. I would choose this option.
